### PR TITLE
Flag added for taking Epistemic mean instead of sum

### PR DESCRIPTION
**Pull request by finalelement**
Prior to this fix, the epistemic entropy based score was directly being used by taking the sum across the image, which gets affected by the size of the image.

The latter can play a role in selection of sample, however it is users choice to alternatively select the mean of entropy which will allow for removal of the effect of the size of the image.

Signed-off-by: vnath <vnath@nvidia.com>

### CODE REVIEW

**Comment by SachidanandAlle** [comment]
CI build is failing ..

**Comment by SachidanandAlle** [comment]
How computing mean among entropy across multiple runs but for the same image helps?

File: monailabel/tasks/scoring/epistemic.py [hunk 1]
```diff
@@ -29,14 +29,17 @@ class EpistemicScoring(ScoringMethod):
     First version of Epistemic computation used as active learning strategy
     """
 
-    def __init__(self, model, network=None, transforms=None, roi_size=(128, 128, 64), num_samples=10):
+    def __init__(
+        self, model, network=None, transforms=None, roi_size=(128, 128, 64), num_samples=10, mean_score_flag=False
+    ):
```
Comment (SachidanandAlle) [inline monailabel/tasks/scoring/epistemic.py:34]: We want this to enabled by default?

File: monailabel/tasks/scoring/epistemic.py [hunk 2]
```diff
@@ -29,7 +29,9 @@ class EpistemicScoring(ScoringMethod):
     First version of Epistemic computation used as active learning strategy
     """
 
-    def __init__(self, model, network=None, transforms=None, roi_size=(128, 128, 64), num_samples=10):
+    def __init__(
+        self, model, network=None, transforms=None, roi_size=(128, 128, 64), num_samples=10, mean_score_flag=False
```
Comment (SachidanandAlle) [inline monailabel/tasks/scoring/epistemic.py:33]: , mean_score_flag=False is not needed anymore

**Review by SachidanandAlle — APPROVED** [review]

**Comment by finalelement** [comment]
@SachidanandAlle hit merge, its blocked at my end.

**Pull request merged**